SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -63,6 +64,7 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 
 	FReceptionistConfig ReceptionistConfig;
 	FLocatorConfig LocatorConfig;
+	FDevAuthConfig DevAuthConfig;
```

Comment:
The following comment is not directly related to this PR: do I understand it right, that we normally don't use 3 configs at once, but just one of them, depending on the context? Would that make sense to have just one pointer to their base class, that's initialised with the relevant config?

Reply:
yes, that is correct. This is something that also confused me when I started looking into it. I agree that it would make sense to have a pointer to their base class and I would really like to change this. I would prefer to do this in a separate PR though to keep them all more or less small and easy to review. :)

Reply:
Separate PR sounds good, could you create a jira ticket and add a `// TODO: UNR-<tickerNumber>` here?

Reply:
added the line. A ticket already exists as part of changing parts of the SpatialWorkerConnection for the dev auth flow